### PR TITLE
Apply contribution guidelines to "Uppercase" rule

### PR DESCRIPTION
**Pull request by DaniloBenevides**
I made the changes requested in  #921 
if something is wrong feel free to contact me.

### CODE REVIEW

**Comment by DaniloBenevides** [comment]
Sorry , I just saw that a lot of files came with my commit how can i fix it ?

**Comment by koficodes** [comment]
@DaniloBenevides  it looks like there were changes to files you weren't supposed to work on. With reference to the guide lines. There should be changes to approximately 4 files.

-rule file
-ruleTest file
-ruleException file
-rule.phpt file
----
you can check if your editor is not changing the indentations of the files. Also try running php-cs-fixer as directed by the contribution guide.

**Comment by DaniloBenevides** [comment]
@koficodes  Thank you for the advices ! So I close this one and open another PR ?
Sorry for the starter questions .

**Comment by koficodes** [comment]
@DaniloBenevides I guess so. I see you changed the base branch to master, that'ts good.

After working on your changes run `git status` to see the files you changed to be sure.

File: library/Exceptions/UppercaseException.php [hunk 1]
```diff
@@ -13,8 +13,16 @@
 
 namespace Respect\Validation\Exceptions;
 
-class UppercaseException extends ValidationException
+/**
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Jean Pimentel <jeanfap@gmail.com>
+ * @author Danilo Benevides <danilobenevides01@gmail.com>
```
Comment (henriquemoody) [inline library/Exceptions/UppercaseException.php]: Please sort it alphabetically

File: library/Rules/Uppercase.php [hunk 2]
```diff
@@ -13,8 +13,19 @@
...
 namespace Respect\Validation\Rules;
 
-class Uppercase extends AbstractRule
+/**
+ * Validates if the input is uppercase.
+ *
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Jean Pimentel <jeanfap@gmail.com>
+ * @author Danilo Benevides <danilobenevides01@gmail.com>
+ * @author Alexandre Gaigalas <alexandre@gaigalas.net>
```
Comment (henriquemoody) [inline library/Rules/Uppercase.php]: Please sort it alphabetically

File: tests/unit/Rules/UppercaseTest.php [hunk 3]
```diff
@@ -13,58 +13,48 @@
...
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\Uppercase
  * @covers \Respect\Validation\Exceptions\UppercaseException
+ *
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Danilo Benevides <danilobenevides01@gmail.com>
+ * @author Jean Pimentel <jeanfap@gmail.com>
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
```
Comment (henriquemoody) [inline tests/unit/Rules/UppercaseTest.php]: Please sort it alphabetically

File: tests/unit/Rules/UppercaseTest.php [hunk 4]
```diff
@@ -13,58 +13,48 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
```
Comment (henriquemoody) [inline tests/unit/Rules/UppercaseTest.php]: There is an extra space here

**Comment by DaniloBenevides** [comment]
@henriquemoody Thank you for the review and the text recommendation I will fix the topics and add the PHPT file.

**Pull request closed without merging**